suite/tests/linux/tool.drcov.eintr.templatex

```diff
@@ -40,7 +40,7 @@ DA:121,1
 DA:122,1
 DA:124,1
 DA:125,1
-DA:126,1
+DA:126,[01]
```

Comment:
Line 105 in `eintr.c` has a very similar loop so it might suffer from a similar race condition. Should we update the expectation for that line too to avoid a repeat of this issue with that line?

Reply:
Definitely, thanks for highlighting that.